query/src/storages/fuse/meta/v1/block.rs

```diff
@@ -1,4 +1,4 @@
-//  Copyright 2021 Datafuse Labs.
+//  Copyright 2022 Datafuse Labs.
```

Comment:
See that we make a new `v1` directory for the v1 version, if we only add a new field of the block layout, how about the new directory?
v2/block.rs or ?

Reply:
yeah, there is another versioned mod (but is v0 :D). will be pushed later